src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -125,6 +125,16 @@ class BaseOptionsSelector extends Component {
     }
 
     componentDidUpdate(prevProps) {
+        if (this.props.autoFocus && !prevProps.isFocused && this.props.isFocused) {
+            InteractionManager.runAfterInteractions(() => {
+                // Focus text input
+                if (!this.textInput) {
+                    return;
+                }
```

Comment:
Should we move this to the top of the function? If we don't have a textInput, we don't need the if, or the call to InteractionManager, right